INN/Largo
# #1870 - Fix landing page options not showing their true saved value

## Changes

This pull request makes the following changes:

- Updates `maybe_unserialize( $fields['show'] )` to `maybe_unserialize( $fields['show'][0] )` inside of the `cftl_tax_landing_main` function.

## Why



For #1870 

## Testing/Questions

Features that this PR affects:

- Series landing page options


Questions that need to be answered before merging:


Steps to test this PR:

1. Create a test series
2. Create a test landing page for your series
3. Check some of these options on the landing page editor and refresh the page. Make sure the options are still checked.

<img width="944" alt="Screen Shot 2021-04-01 at 6 14 25 PM" src="https://user-images.githubusercontent.com/18353636/113359648-36082880-9316-11eb-8648-471c4524efca.png">
